docs/continuing_runs.md

```diff
@@ -0,0 +1,225 @@
+# Continuing and editing runs
+
+The modularity of HADDOCK3 allows users to take successful steps
+(modules) of a workflow as starting points for new runs. There are
+several ways users can continue or expand previous runs. The index below
+lists those possibilities:
+
+1. Restart, editing, and extending a previous run from a given step
+1. Copy a run to a new folder, and continue from there
+    1. Use a single step as a starting point for a new run
+1. Extend a run with new modules
+1. Final considerations
+
+# Restart, editing, and extending a previous run from a given step
+
+This option allows you to restart a run from a specific step. It is
+useful when you want to repeat part of the run or to continue a broken
+run without needing to rerun all successful steps.
+
+Let's imagine you execute a run with six steps (modules).
+
+```
+haddock3 my-haddock3-config-file.cfg
+```
+
+If the run completes successfully, the run directory shows as follows:
+
+```
+0_topoaa/
+1_rigidbody/
+2_caprieval/
+3_seletop/
+4_flexref/
+5_caprieval/
+data
+log
+```
+
+Now, imagine that later in your analysis, you notice that you used the
+wrong parameters for the flexible refinement (`4_flexref`) stage,
+setting `randremoval=true` instead of `randremoval=false`. Therefore,
+you want to repeat the run only for `4_flexref` onwards without
+repeating the previous successful steps. For that, edit the
+configuration file according to your new preferences, and run:
+
+```
+haddock3 my-haddock3-config-file.cfg --restart 4
+```
+
+This operation will delete modules' folders `4_` onwards and repeat
+those steps following the configurations in the (new) configuration
+file. You must run the `haddock3` command from within the same directory
+as you did for the original run.
+
+As you can see, this option allows you to **restart** or **repeat** a
+run from a specific step using the **original** configuration file or to
+**modify** a run by updating the parameters for the steps you want to
+rerun.
```

Comment:
This is still confusing, if you use the original you restart, if you use the modified you repeat? What is the difference between restart and repeat or here is just a synonymy?